pom.xml

```diff
@@ -67,7 +67,7 @@
 
   <!-- Bring some sanity to version numbering... -->
   <properties>
-    <jenkins.version>2.222.4</jenkins.version>
+    <jenkins.version>2.321</jenkins.version>
```

Comment:
Shouldn't we favor targeting the same Jenkins release as before (i.e. leaving this unchanged) and use the Jenkinsfile/build for testing on the 2.321 too, instead?

Reply:
Personally I tend to prefer keeping it simple and just bumping the dep. If and when someone wants some unrelated bug fix backported for use on older lines, it is trivial to do that.

Reply:
Makes sense I guess. In particular with such plugins where it tends to not get tons of updates, this is probably simpler indeed.

Reply:
Yeah, that makes sense.